src/select.coffee

```diff
@@ -64,10 +64,10 @@ Ember.AddeparMixins.ResizeHandlerMixin,
   placeholder:        undefined
   disabled: no
 
-  # we need to set tabindex so that div responds to key events
   highlightedIndex: -1
 
-  tabindex: -1
+  # we need to set tabindex so that div responds to key events
+  tabindex: 0
```

Comment:
this change is good, but you should add `tabindex: -1` to the MultiSelectComponent, because right now you have to press tab twice (once for the component and once for the text field inside). I just tested with `tabindex: -1` and it seems to work as expected.